src/test/java/teammates/test/cases/browsertests/InstructorFeedbackResultsPageUiTest.java

```diff
@@ -702,7 +702,7 @@ private void testDownloadAction() {
                                               .withUserId("CFResultsUiT.instr");
         browser.driver.get(reportUrl.toAbsoluteString());
         String afterReportDownloadUrl = browser.driver.getCurrentUrl();
-        assertFalse(reportUrl.equals(afterReportDownloadUrl));
+        assertFalse(reportUrl.toString().equals(afterReportDownloadUrl));
```

Comment:
Currently,  we don't have assertNotEquals method, so you can revert the last commit. Sorry for confusion.